moonbeam-types-bundle/package.json

```diff
@@ -6,6 +6,7 @@
   "types": "dist/index.d.ts",
   "prepublish": "tsc",
   "scripts": {
+    "compare": "rm -fr dist && tsc && npx ts-node check.ts",
```

Comment:
What is this doing I don't get it

Reply:
this is running the check script. rebuilding the bundle's js version is necessary everytime